src/App.tsx

```diff
@@ -1,28 +1,26 @@
-import React from "react";
-import Particles from "react-particles-js";
-import "./styles/styles.css";
-import "animate.css/animate.min.css";
-import "bootstrap/dist/css/bootstrap.min.css";
-import TopNavBar from "./components/TopNavBar.jsx";
-import Hero from "./components/Hero";
-import Features from "./components/Features.jsx";
-import Team from "./components/Team.jsx";
-import Contribute from "./components/Contribute.jsx";
+import React from 'react';
+import Particles from 'react-particles-js';
+import './styles/styles.css';
```

Comment:
While you converted all the styling over to the scss file, your still importing the css file and thus, drawing styles from it instead of the scss.